documentation/DATA_FORMAT.md

```diff
@@ -589,24 +598,33 @@ For example, if a dataset had the following tracks and outliers, the file might
 
 ## Collections
 
-Collections are defined by an optional JSON file and group one or more datasets for easy access. Timelapse-Colorizer can parse collection files and present its datasets for easier comparison and analysis from the UI.
+Collections are defined by an optional JSON file and group one or more datasets together. Timelapse-Colorizer can parse collection files and present its datasets for easier comparison and analysis from the UI.
 
 By default, collection files should be named `collection.json`.
 
-Collections are an array of JSON objects, each of which define the `name` (an **alias**) and the `path` of a dataset. This can either be a relative path from the location of the collection file, or a complete URL.
-
-If the path does not define a `.json` file specifically, Timelapse-Colorizer will assume that the dataset's manifest is named `manifest.json` by default.
-
 `collection.json:`
 
 ```txt
-[
-    { "name": <some_name_1>, "path": <some_path_1>},
-    { "name": <some_name_2>, "path": <some_path_2>},
-    ...
-]
+{
+    "datasets": [
+        { "name": <some_name_1>, "path": <some_path_1>},
+        { "name": <some_name_2>, "path": <some_path_2>},
+        ...
+    ],
+    "metadata": {
```

Comment:
this looks odd to me, I think I would remove the "metadata" layer and just put all those data fields at the top level.  We can discuss that if you want.

Reply:
I'd like to keep the metadata as its own file so I can share the logic that updates the dataset and collection metadata (updating revision number, date last modified, creation date, etc.). I imagine it'll be a little easier to reason about both that way.

Reply:
I didn't mean to change the breakdown of files. I meant that within this collection.json file I wouldn't have a "metadata" sub-object.  I'd just promote the metadata fields to be DATA FIELDS of the COLLECTION.  At the same hierarchichal level as `"datasets"`

Reply:
The main reason is: what is metadata?  In this sense it just feels like an artificial grouping.  Would there be any fields that are not metadata?

Reply:
Oops, I mistyped! Meant to say keeping the metadata as its own field, so yes we're on the same page. 

Do you think it would be strange to have that be inconsistent between the dataset and collection files? The dataset manifests have metadata stored under the `metadata` field currently.

Reply:
Posted this before I saw your updated comment! The way we have it in the dataset JSON, the root level fields are all references to files or lists of file-related descriptors; anything that's not pointing to file paths is stored under the metadata field.

I don't expect we'll add additional files to the collection schema (besides maybe a markdown description file or something in the future?), but it feels reasonable to me to be consistent.

Reply:
After more discussion, it still looks odd to me, but much less odd.  I'm good to go with it!